R/txrange.R

```diff
@@ -0,0 +1,265 @@
+#' Assigns each read class geneScore and txScore
+#' @param se summerized experiment object with read classes/ranges
+#' @param genomeSequence genomeSequence
+#' @param annotations GRangesList of annotations
+txrange.scoreReadClasses = function(se, genomeSequence, annotations, 
+                                    min.readCount = 2){
+    options(scipen = 999)
```

Comment:
Yes this line is required because of the automatic scientific notation of bases such as 900000. It is also called a few times within the main bambu code, but is then set back to 0. If I do not have this line it will crash on some samples. Theoretically we should be able to set this once at the start of the code but perhaps there was a reason Chen Ying changes it to 0 at some points?

Reply:
yes this is related to converting integers to characters. It's not a good style to change global options that is why we changed it back I think. Maybe a better way would be to use something like `format(i, scientific = FALSE)`. Mabye we leave it here since there was no complaing from bioconductor about this? You could add a comment on why this is needed here

Reply:
small change in 5e19452